### ISSUE TITLE
Missing $STRUCTS section in profiles created using parse_pdb command

### ISSUE THREAD

**Issue by fwruck** [google/rekall#355]
Hi,
this issue was originally reported on the libvmi github page libvmi/libvmi#548, however it seems to be a rekall issue, so i moved my comment.

The issue is that no $STRUCTS section is contained in the output of rekall parse_pdb.
The pdb I'm using is the file ntkrnlmp.pdb, PDB GUID: 9378084E8DBD4AB1A155099BCE693E341
I'm using Debian stretch.

I did the following tests:

    Obtain the PDB GUID using vmi-win-guid
    Obtain the PDB GUID using the symchk Microsoft tool
    download the pdb-file from msdl.microsoft.com/download/symbols using wget
    download the pdb-file using the symchk tool
    install the newest rekall-version from github using virtualenv (Version 1.7.3.dev7)
    Install the newest rekall version as described on drakvuf.com:

sudo pip install setuptools
python setup.py build
sudo python setup.py install

In each combination of the approaches, the $STRUCTS section is missing.

The only way I got it to work was using the packaged version of rekall (python-rekall-core, pyton-rekall) coming with Debian stretch (1.6.0+dfsg-2).
(If you have local copies you have to make sure that all rekall-scripts are removed from /usr/local/...)

It seems the issue has arised between Versions 1.6.2 and 1.7.3.dev7

**Comment by scudette** [comment]
Sometimes MS release bad pdb files and then they go back and fix them - so it is possible that this is already fixed. I have tried to do the following steps:

```
⟫ rekall -v fetch_pdb ntkrnlmp.pdb 9378084E8DBD4AB1A155099BCE693E341
2018-01-24 10:20:24,870:DEBUG:rekall.1:Logging level set to 10
2018-01-24 10:20:24,871:DEBUG:rekall.1:Running plugin (fetch_pdb) with args (()) kwargs ({'pdb_filename': 'ntkrnlmp.pdb', 'guid': '9378084E8DBD4AB1A155099BCE693E341', 'dump_dir': None, 'verbosity': 1})
 Trying to fetch http://msdl.microsoft.com/download/symbols/ntkrnlmp.pdb/9378084E8DBD4AB1A155099BCE693E341/ntkrnlmp.pdb
⟫ rekal -v parse_pdb ntkrnlmp.pdb > /tmp/9378084E8DBD4AB1A155099BCE693E341.json
2018-01-24 10:20:42,486:DEBUG:rekall.1:Logging level set to 10
2018-01-24 10:20:42,487:DEBUG:rekall.1:Running plugin (parse_pdb) with args (()) kwargs ({'pdb_filename': 'ntkrnlmp.pdb', 'profile_class': None, 'output_filename': None, 'windows_version': None, 'concise': None, 'dump_dir': None, 'verbosity': 1})
2018-01-24 10:20:42,864:DEBUG:rekall.1:Opened url https://github.com/google/rekall-profiles/raw/master/v1.0/inventory.gz
2018-01-24 10:20:42,928:DEBUG:rekall.1:Opened local file /home/mic/.rekall_cache/v1.0/inventory.gz
2018-01-24 10:20:43,001:DEBUG:rekall.1:Opened url http://profiles.rekall-forensic.com/v1.0/inventory.gz
2018-01-24 10:20:43,043:DEBUG:rekall.1:Opened local file /home/mic/.rekall_cache/v1.0/inventory.gz
2018-01-24 10:20:43,043:DEBUG:rekall.1:Opened local file /home/mic/.rekall_cache/v1.0/mspdb.gz
2018-01-24 10:20:43,047:INFO:rekall.1:Loaded profile mspdb from Local Cache Directory:/home/mic/.rekall_cache (in 0.0039958953857421875 sec)
```
and that file contains the proper $STRUCTS:
``` 
"$STRUCTS": {
  "<unnamed-10040>": [8, {
   "IdleTime": [0, ["unsigned long", {}]], 
   "NonIdleTime": [4, ["unsigned long", {}]]
  }], 
```

This is what I fetched from MS:
```
⟫ ls -l ntkrnlmp.pdb 
-rw-rw-r-- 1 mic mic 6884352 Jan 24 10:20 ntkrnlmp.pdb
⟫ md5sum ntkrnlmp.pdb 
eb2ae344390219b44becff7812c57082  ntkrnlmp.pdb 
```

**Comment by fwruck** [comment]
Hi,
the checksum of my pdb file is the same.
Following the exactly same steps as you produces the broken pdb.
However I tested it on the machine of a friend and it worked without problems.

I tried the following without success:
 - Install in virtualenv using pip
 - Install in virtualenv using git
 - checkout 1.7.2rc1
 - checkout 1.7.3.dev7
 - delete the ~/.rekallrc and ~/.rekall_cache files
 - use the ~/.rekallrc and ~/rekall_cache from my friends machine where it is working
 - delete any rekall packages from /usr/local/

The issue seems to be related to my machine,
so I'll close the issue for now.
I'll inform you when I have more information

I have no
